lib/utilities/lint-fix.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const chalk = require('chalk');
+const execa = require('execa');
+
+async function run() {
+  let lintFixScriptName = `lint:fix`;
+  let cwd = process.cwd();
+  let npmRunResult = await execa('npm', ['run'], { cwd });
```

Comment:
We should probably use either npm or yarn based on the normal "is yarn.lock present" trigger that we normally use